src/pytsql/grammar/TSqlParserListener.py

```diff
@@ -1,6 +1,6 @@
-# Generated from TSqlParser.g4 by ANTLR 4.11.1
+# Generated from TSqlParser.g4 by ANTLR 4.13.1
 from antlr4 import *
-if __name__ is not None and "." in __name__:
+if "." in __name__:
```

Comment:
I'd suggest excluding the generated code from formatting/linting.

Reply:
I'm confused about why that line changed.

I guess that for now, you can forget then about the exclusion.